packages/synapse-react-client/test/testutils/TestingLibraryUtils.tsx

```diff
@@ -10,6 +10,12 @@ type RtlWrapperProps = {
   children?: ReactNode
 }
 
+const reactQueryTestConfig = {
+  ...defaultQueryClientConfig,
+  // Recommended to set cacheTime to Infinity: https://tanstack.com/query/v3/docs/react/guides/testing#set-cachetime-to-infinity-with-jest
+  cacheTime: Infinity,
+}
```

Comment:
Just an optimization to prevent the garbage collector from inadvertently failing tests. I have yet to actually see it happen, but the react-query docs recommend it.

We use a new QueryClient for each test, and the default cacheTime is much longer than any particular test would run (5min), so this should not impact on our tests otherwise.